ambry-router/src/main/java/com/github/ambry/router/GetBlobInfoOperation.java

```diff
@@ -439,6 +439,9 @@ private void checkAndMaybeComplete() {
     if (progressTracker.isDone()) {
       if (progressTracker.hasSucceeded()) {
         operationException.set(null);
+      } else if (operationTracker.maybeFailedDueToOfflineReplicas()) {
```

Comment:
Why are we using this logic for GetBlobInfo/GetBlob operations too?

Reply:
removed.